src/main/java/com/maxmind/geoip2/record/Traits.java

```diff
@@ -146,6 +178,21 @@ public ConnectionType getConnectionType() {
         return this.connectionType;
     }
 
+    /**
+     * @return The static IP score of the IP address. This is an indicator of
+     * how static or dynamic an IP address is. This attribute is only
+     * available from GeoIP2 Precision Insights.
+     */
+    @JsonProperty("static_ip_score")
+    public Double getStaticIPScore() {
```

Comment:
This should be `getStaticIpScore` to be consistent with our naming for our Java libraries.